frontend/taipy/package.json

```diff
@@ -38,7 +38,8 @@
     "fast-deep-equal": "^3.1.3",
     "formik": "^2.2.9",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "taipy-gui": "file:../../taipy/gui/webapp"
```

Comment:
this should not be committed
as you're not touching taipy, please do not commit anything in that folder